jekyll/_cci2/configuration-reference.md

```diff
@@ -275,31 +275,35 @@ The [machine executor]({{ site.baseurl }}/2.0/executor-types) is configured by u
 
 Key | Required | Type | Description
 ----|-----------|------|------------
-enabled | N | Boolean | This must be true in order to enable the `machine` executor.  Is required if no other value is specified
-image | N | String | The image to use (default: `circleci/classic:latest`). **Note:** This key is **not** supported on the installable CircleCI. For information about customizing `machine` executor images on CircleCI installed on your servers, see our [VM Service documentation]({{ site.baseurl }}/2.0/vm-service).
+image | Y | String | The VM image to use. View [available images](#available-machine-images). **Note:** This key is **not** supported on the installable CircleCI. For information about customizing `machine` executor images on CircleCI installed on your servers, see our [VM Service documentation]({{ site.baseurl }}/2.0/vm-service).
 docker_layer_caching | N | Boolean | Set to `true` to enable [Docker Layer Caching]({{ site.baseurl }}/2.0/docker-layer-caching). **Note:** You must open a support ticket to have a CircleCI Sales representative contact you about enabling this feature on your account for an additional fee.
 {: class="table table-striped"}
 
-As a shorthand, you can set the `machine` key to `true`.
 
 Example:
 
-``` YAML
+```yaml
+version: 2.1
+workflows:
```

Comment:
There's no need to add this workflow in 2.1 config - it will get added automatically